politeiawww/cmd/politeiawww_dbutil/politeiawww_dbutil.go

```diff
@@ -159,7 +165,8 @@ const usageMsg = `politeiawww_dbutil usage:
     -migrate
           Migrate from one user database to another
           Required DB flag : None
-          Args             : <fromDB> <toDB>
+          Args             : <fromDB> <mysql/cockroachdb/leveldb>
```

Comment:
This reads weird to me.  Might something like this be cleaner?

```
          Args             : <fromDB> <toDB>
                             Valid DBs are mysql, cockroachdb, leveldb
```

Reply:
👍🏽